src/vstdlib/strtools.cpp

```diff
@@ -402,6 +402,10 @@ void V_SplitString2( const char *pString, const char * const *pSeparators, int n
 			{
 				outStrings.AddToTail( AllocString( pCurPos, -1 ) );
 			}
+			else if (bIncludeEmptyStrings)
+			{
+				outStrings.AddToTail( AllocString( "" , 1 ));
```

Comment:
I think a similar check for bIncludeEmptyStrings needs to exist for the first separator.  Otherwise a string like ",second,third" would not result in an empty first entry as it should.

But don't fix that yet.  It bothers the heck out of me that there is a bug in this function to begin with.  I'm going to investigate what's going on in the Steam branch.